From 036ec29c708e5cb75afef9e895263ca64ba7c795 Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Wed, 27 Feb 2019 00:20:06 +0200
Subject: [PATCH 1/3] Messaging: Fix generating id in android namespace

Fixes:
res/layout/apn_preference_layout.xml:40: warn:
generated id 'android:id/title' for external package 'android'.
res/layout/apn_preference_layout.xml:48: warn:
generated id 'android:id/summary' for external package 'android'.

Change-Id: Ia096fa2e072ba4391e3adad52d3f9c9abed0e976
---
 res/layout/apn_preference_layout.xml                        | 4 ++--
 src/com/android/messaging/ui/appsettings/ApnPreference.java | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/res/layout/apn_preference_layout.xml b/res/layout/apn_preference_layout.xml
index 25a0323..8b864f6 100644
--- a/res/layout/apn_preference_layout.xml
+++ b/res/layout/apn_preference_layout.xml
@@ -38,7 +38,7 @@
         android:background="?android:attr/selectableItemBackground">
 
         <TextView
-            android:id="@+android:id/title"
+            android:id="@+id/title"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:focusable="true"
@@ -46,7 +46,7 @@
             android:textAppearance="?android:attr/textAppearanceListItem" />
 
         <TextView
-            android:id="@+android:id/summary"
+            android:id="@+id/summary"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_below="@android:id/title"
diff --git a/src/com/android/messaging/ui/appsettings/ApnPreference.java b/src/com/android/messaging/ui/appsettings/ApnPreference.java
index 74c6a08..c5cc85a 100644
--- a/src/com/android/messaging/ui/appsettings/ApnPreference.java
+++ b/src/com/android/messaging/ui/appsettings/ApnPreference.java
@@ -94,7 +94,7 @@ public class ApnPreference extends Preference implements
 
     public void setApnRadioButtonContentDescription(final CompoundButton buttonView) {
         final View widget = (View) buttonView.getParent();
-        final TextView tv = (TextView) widget.findViewById(android.R.id.title);
+        final TextView tv = (TextView) widget.findViewById(R.id.title);
         final String apnTitle = tv.getText().toString();
         buttonView.setContentDescription(apnTitle);
     }
-- 
2.17.1

